index.js

```diff
@@ -73,25 +82,51 @@ FeatureParser.prototype.getTestFeature = function() {
       ];
       break;
     default: // qunit
-      testFeature = [
-        "function testFeature(feature) {",
-        "  module(`Feature: ${feature.title}`, {",
-        "    beforeEach: function() {",
-        "      this.application = startApp();",
-        "    },",
-        "    afterEach: function() {",
-        "      Ember.run(this.application, 'destroy');",
-        "    }",
-        "  });",
+      if(unitModule && unitModule.indexOf('component') === 0){
+        testFeature = [
+          "function testFeature(feature) {",
+          "  moduleForComponent('"+fileName+"', `Feature: ${feature.title}`,{",
+          "     unit:true,",
+          "     needs: feature.annotations.needs? feature.annotations.needs.split(',') : [],",
+          "});",
+
+          "  feature.scenarios.forEach(function(scenario) {",
+          "    test(`Scenario: ${scenario.title}`, function(assert) {",
+          "      expect(scenario.steps.length);",
+          "      let self = this;",
+          "      return new Ember.RSVP.Promise(function(resolve){ yadda.Yadda(library.default(assert), self).yadda(scenario.steps, { ctx: {} }, resolve); });",
+          "    });",
+          "  });",
+          "};"
+        ];
+      }else if(unitModule && unitModule.indexOf('model') === 0){
+        // TODO: Implement unit test for models
```

Comment:
maybe a good idea to write something friendly to the command line when someone uses it :smile: